### DISCUSSION TITLE
Why a PoW consus is less decentralized than a PoS

### DISCUSSION THREAD

**Discussion by wilonweb** [smartcontractkit/full-blockchain-solidity-course-py#424] [Q&A] [answered]
Hello World. 

Do you know why PoW consensus is less decentralized than PoS consensus ? 
Is it because PoW has a flaw with sybill attack and PoS  has a flaw with attack 51 ?

or both have a flaw with these two attacks ?

**Comment by cromewar** [comment, marked as answer]
Hello @NinjaTurtles-cloud, the decentralization depends on more factors, as if the project has governance, uses oracles, if is linked to an specific company and so on... but talking about just to PoS and PoW, well pow requires to invest money on computers with powerful graphic cards, also miners have to pay high electricity bills.. so you  can say PoW is less decentralized as not everyone can access the equipment needed to become a miner. In the other hand PoS also requires to pay tokens fees as pledge to ensure the node is going to behave correctly, so it is also not 100% decentralized,

I highly recommend you to take a look to [this](https://cointelegraph.com/blockchain-for-beginners/proof-of-stake-vs-proof-of-work:-differences-explained) to understand deeply how each one works.

Note: both are vulnerable to attacks depending on the context.

**Reply by wilonweb** [reply]
Thanks you !!!
